samples/gui/Deployment.yaml

```diff
@@ -56,12 +56,12 @@ spec:
     spec:
       containers:
       - name: datauserclient
-        image: image-registry.openshift-image-registry.svc:5000/datauser1/datauserclient:latest
+        image: image-registry.openshift-image-registry.svc:5000/default/datauserclient:latest
         imagePullPolicy: Always
```

Comment:
For integration with Watson Studio it appears that we will need to deploy to namespace cp4d, which is where the user's compute runs.  If we have a deployment script, this should be a parameter of the deployment.

Reply:
Done.